### ISSUE TITLE
Specific information to recreate leaderboard results?

### ISSUE THREAD

**Issue by Bluffish** [Megvii-BaseDetection/BEVDepth#20]
Hello, I have been struggling to recreate the results shown on the detection leaderboard. Can someone provide me with the specific information to recreate?

**Comment by yinchimaoliang** [comment]
Hi! The input resolution is 640 x 1600, the backbone we use is ConvNext, the bev resolution is 256 x 256, we also use train and val split for training.

**Comment by Bluffish** [comment]
Hi, thanks for the info! I'll make sure to give it a try!

**Comment by Bluffish** [comment]
Hello, can you tell me which ConvNeXt architecture you guys use? Tiny, small base or large? Thanks!

**Comment by zehuichen123** [comment]
Hi, @yinchimaoliang, I wonder if the data augmentation in your submission is the same one used in the r50 config? So the only differences are (I) image resolution (2) bev resolution (3) image backbone (4) trainval for test, right?

**Comment by yinchimaoliang** [comment]
> 

Hi! When the resolution of the input image is changed, the `resize_lim` and `final_dim` of ida should be changed accordingly.

**Comment by swtju14** [comment]
Hi, @Bluffish. BEVDepth-pure extends the BEVDepth by using ConvNeXt-base as backbone.

**Comment by zehuichen123** [comment]
Hi, @yinchimaoliang thanks for your information. I am trying to use ConvNeXt but encountered OOM problem. I wonder if you adopt torch.utils.checkpoint to save your memory in the ConvNeXt backbone? If so, do you use it in the stage conv or also in the downsample layers ?Thanks!

**Comment by yinchimaoliang** [comment]
Hi, @zehuichen123 , we didn't adopt torch.utils.checkpoint, we used V100 with a memory of 32GB, you can try it if you are using machines with smaller memory size.

**Comment by zehuichen123** [comment]
Thanks! I am using V100 machine too. BTW, how did you set your batch size for each GPU as well as the learning rate? (since a batch size of 8 is impossible in this case)

**Comment by yinchimaoliang** [comment]
> Thanks! I am using V100 machine too. BTW, how did you set your batch size for each GPU as well as the learning rate? (since a batch size of 8 is impossible in this case)

The batch size for each card is set to 2, and we use 4 machines. Learning rate is 2e-4.

**Comment by zehuichen123** [comment]
Got it! Thanks! 😊

**Comment by zehuichen123** [comment]
Hi @yinchimaoliang , I am trying with ConvNeXt-base on the A100(40G) but I found that even with batch size 1 I still encounter OOM error. The code can be successfully run with 256x708 scale, but failed with 640x1600. :(

**Comment by fcjian** [comment]
@yinchimaoliang Do you have any plan to release the config file and model shown on the detection leaderboard?

**Comment by fcjian** [comment]
Hi, @zehuichen123, we also encountered OOM problem. Did you solve this problem？

**Comment by zeyuwang615** [comment]
> > Thanks! I am using V100 machine too. BTW, how did you set your batch size for each GPU as well as the learning rate? (since a batch size of 8 is impossible in this case)
> 
> The batch size for each card is set to 2, and we use 4 machines. Learning rate is 2e-4.

@yinchimaoliang Did you use base lr = 2e-4 or final lr = 2e-4? I see in the code the basic learning rate for 64 samples per batch is set to 2e-4. I wonder if it is the same for 2 (batch size) * 4 (machine) * 8(gpu per machine)?

**Comment by zeyuwang615** [comment]
Also, I see from [here](https://github.com/Megvii-BaseDetection/BEVDepth/issues/16#issuecomment-1200116691) that you uses dcn head in the submitted version to the test set leaderboard, which is not mentioned in this issue. I wonder if there is anything else we should pay attention to to reproduce the leaderboard results (other than (I) image resolution (2) bev resolution (3) image backbone (4) use both train/val set for training)?

**Comment by zeyuwang615** [comment]
Also, I have noticed ConvNeXT uses different `drop_path_rate` for different architectures. What value did you use for `drop_path_rate`?

**Comment by pummi823** [comment]
Why does the author not answer these questions （by zeyuwang615）？？ They are quite important T T
